source/about/cloud-subscriptions.rst

```diff
@@ -61,8 +65,17 @@ After your first month, the billing period begins at 12 AM UTC on the first day
 - Similarly, if you end your subscription in the middle of a billing period, charges will be prorated. For example, if you cancel your account on 6/15, we'll charge you on 7/1 for the period 6/1 - 6/15.
 - If you upgrade in the middle of a billing period, charges will also be prorated. For example, if you upgrade from Starter to Professional on 6/15, we'll charge you 7/1. This will be a single charge for Starter for the period 6/1 - 6/14, then a separate charge for Professional for the period 6/15 - 6/30.
 
-Can I purchase an annual subscription to Mattermost Cloud deployments?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+What happens if I decide to purchase a subscription to Mattermost Enterprise during my trial?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Your trial will continue and on day 31 billing will commence. Depending on what day of the month this is, you'll be billed pro-rata. You'll receive your bill at the end of the calendar month for this month and going forward.
+
+How do I change my subscription from Mattermost Enterprise to Mattermost Professional?
```

Comment:
@jasonblais, I'll need your input for this. :)

Reply:
@johndavidlugtu deferring to you :)